data/features/m/m.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+m() {
+  if command -v man >/dev/null; then
+    if [[ $# -eq 1 ]]; then
+      man_output=$(man "$@" | grep -E '^ +-[a-zA-Z], --[a-zA-Z-]+')
+      echo "$man_output"
```

Comment:
We use curly brackets to expand variables. You did not know that right? ...

Reply:
solved